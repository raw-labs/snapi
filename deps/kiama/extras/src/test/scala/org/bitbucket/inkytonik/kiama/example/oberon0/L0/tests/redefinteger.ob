MODULE redefinteger;

TYPE
    INTEGER = BOOLEAN;

VAR
    a : INTEGER;

BEGIN
    a := TRUE
END redefinteger.
