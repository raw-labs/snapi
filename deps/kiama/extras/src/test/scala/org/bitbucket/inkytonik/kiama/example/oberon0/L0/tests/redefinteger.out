* _pp.ob
MODULE redefinteger;

    TYPE
        INTEGER = BOOLEAN;

    VAR
        a : INTEGER;

BEGIN
    a := TRUE
END redefinteger.
[6]* c
[6]#include <stdio.h>
[6]typedef int ob_INTEGER2;
[6]ob_INTEGER2 ob_a3;
[6]int main (int argc, char *argv[0]) {
[6]    ob_a3 = 1;
[6]    return 0;
[6]}
